docs/_data/navigation.yml

```
@@ -3,7 +3,7 @@ rse:
     children:
     - title: "Information for new RSEs"
       url: /roles/new-rse/#resources-and-information-for-new-rses
-- title: "What is an RSE?"
+  - title: "What is an RSE?"
```

Looks like two spaces are still missing...

```suggestion
    - title: "What is an RSE?"
```